utilities/general.py

```diff
@@ -69,7 +69,7 @@ def download_model_data(
     containers = [
         {
             "name": "model-downloader",
-            "image": "quay.io/modh/kserve-storage-initializer@sha256:330af2d517b17dbf0cab31beba13cdbe7d6f4b9457114dea8f8485a011e3b138",
+            "image": "quay.io/modh/kserve-storage-initializer@sha256:330af2d517b17dbf0cab31beba13cdbe7d6f4b9457114dea8f8485a011e3b138",  # noqa: E501
```

Comment:
this might not be needed due to https://github.com/opendatahub-io/opendatahub-tests/pull/97/files#diff-ac6c824bcc7455fa0967c33f2062f5c5cf7cee2b62d6402eec5b39c359530e02R78

Reply:
yes; just to pass flake8 check in this pr